xapian-letor/configure.ac

```diff
@@ -429,15 +429,6 @@ if test -n "${EATMYDATA-unset}" ; then
   AC_PATH_PROG([EATMYDATA], [eatmydata], [])
 fi
 
-dnl Used by tests/harness/testsuite.cc
-AC_CHECK_FUNCS([sigaction])
-AC_MSG_CHECKING([for sigsetjmp and siglongjmp])
-AC_TRY_COMPILE([#include <setjmp.h>],
-  [sigjmp_buf jb; if (sigsetjmp(jb, 1)) { siglongjmp(jb, 1);  }],
-  AC_DEFINE([HAVE_SIGSETJMP], [1], [Define to 1 if you have the 'sigsetjmp' function])
-  AC_MSG_RESULT([yes]),
-  AC_MSG_RESULT([no]))
-
```

Comment:
All of the things under "Check which database backends should be built." that are actually to do with backends are irrelevant, because they're -core configuring which ones to _build_. You just use whatever's there, so you don't need that.

Visibility, VPATH and zlib you probably still need. Take some time to read through the diff and understand what each section is actually doing — and ask if you can't figure it out :-)